### PR TITLE
feat: The SSRF request timeout configuration item is added

### PR DESCRIPTION
**Pull request by gubinjie**
# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [x] Please open an issue before creating a PR or link to an existing issue
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

# Description
The SSRF request timeout configuration item is added

Fixes 

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [x] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# Testing Instructions

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Comment by gubinjie** [comment]
@BenjaminX   why? Run Pytest / API Tests (3.10)

File: api/core/helper/ssrf_proxy.py [hunk 1]
```diff
@@ -12,6 +12,10 @@
 SSRF_PROXY_HTTP_URL = os.getenv("SSRF_PROXY_HTTP_URL", "")
 SSRF_PROXY_HTTPS_URL = os.getenv("SSRF_PROXY_HTTPS_URL", "")
 SSRF_DEFAULT_MAX_RETRIES = int(os.getenv("SSRF_DEFAULT_MAX_RETRIES", "3"))
+SSRF_DEFAULT_TIME_OUT = float(os.getenv("SSRF_DEFAULT_TIME_OUT", "5"))
+SSRF_DEFAULT_CONNECT_TIME_OUT = float(os.getenv("SSRF_DEFAULT_CONNECT_TIME_OUT", "5"))
```
Comment (crazywoola) [inline api/core/helper/ssrf_proxy.py:16]: I think this should be a integer.

Comment (gubinjie) [reply api/core/helper/ssrf_proxy.py:16]: @crazywoola  The official httpx documentation uses 10.0, so I use float.
https://www.python-httpx.org/advanced/timeouts/

Comment (gubinjie) [reply api/core/helper/ssrf_proxy.py:16]: image

**Review by crazywoola — APPROVED** [review]

**Comment by laipz8200** [comment]
Would you mind opening another PR to add those new environment variables to the `.env.example` file under the `docker/` dir?

**Comment by gubinjie** [comment]
Don't mind

**Pull request merged**